src/System.Net.Requests/tests/Configurations.props

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <BuildConfigurations>
       netstandard;
+      uap-Windows_NT;
```

Comment:
System.Net.Requests depends on System.Net.Http and depends on System.Net.Primitives  And with #20947 using a separate 'uap' build for System.Net.Primitives, I wanted to make sure it used the right binaries.